libwaysip/src/lib.rs

```diff
@@ -1,625 +1,40 @@
+mod dispatch;
 mod error;
 mod render;
+pub mod state;
+
```

Comment:
```rust
pub use state::WaysipKind;
```
will it be better to add this line?

Reply:
Sure, again this PR is incomplete so please bear with me 😊. Will let you know once I iron out all the kinks